lms/static/sass/elements/_dropdown.scss

```diff
@@ -0,0 +1,105 @@
+.wrapper-more-actions {
+    @include float(right);
+    @include margin-right(($baseline/2));
+    display: inline-block;
+    position: relative;
+
+    .button-more.has-dropdown {
+        @extend %ui-depth2;
+        position: relative;
+        height: ($baseline*2);
+        width: ($baseline*2);
+        padding: 0;
+        border: 1px solid transparent;
+        background: $white;
+        box-shadow: none;
+        font-size: $body-font-size;
+        text-shadow: none;
```

Comment:
Is this needed to reset some bad properties set elsewhere in the LMS? If so, please add a comment noting that for any rules that are defined for that purpose, please.